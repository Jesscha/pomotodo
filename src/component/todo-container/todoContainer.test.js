import React from 'react';
import { shallow } from 'enzyme';
import {TodoContainer} from './todoContainer.component';


describe("<TodoContainer/ >", ()=>{
    const wrapper = shallow(<TodoContainer todoItems={[]}  />)


    it("renders InpunContainer", ()=>{
        expect(wrapper).toMatchSnapshot();
    });

    



    
    })





