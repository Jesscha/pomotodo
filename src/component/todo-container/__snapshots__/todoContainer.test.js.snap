// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoContainer/ > renders InpunContainer 1`] = `ShallowWrapper {}`;
